src/AppEndToEnd.test.tsx

```diff
@@ -253,27 +259,30 @@ it('basic end-to-end flow', async () => {
   getByText(/Vote for 4/i)
 
   // Select first candidate
```

Comment:
This comment seems incorrect. It's selecting the first _and_ second candidates, right?

Reply:
Previous copy/pasta by me!